endoreg_db/models/media/video/video_processing.py

```
@@ -0,0 +1,184 @@
+    # Operation Types
+    OPERATION_MASKING = 'mask_overlay'
+    OPERATION_FRAME_REMOVAL = 'frame_removal'
+    OPERATION_ANALYSIS = 'analysis'
+    OPERATION_REPROCESSING = 'reprocessing'
+    
+    OPERATION_CHOICES = [
+        (OPERATION_MASKING, 'Mask Overlay'),
+        (OPERATION_FRAME_REMOVAL, 'Frame Removal'),
+        (OPERATION_ANALYSIS, 'Sensitivity Analysis'),
+        (OPERATION_REPROCESSING, 'Full Reprocessing'),
+    ]
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)